src/Features/Onboarding.php

```diff
@@ -311,14 +311,51 @@ public function component_settings( $settings ) {
 
 		// Only fetch if the onboarding wizard is incomplete.
 		if ( $this->should_show_profiler() ) {
-			$settings['onboarding']['productTypes'] = self::get_allowed_product_types();
-			$settings['onboarding']['themes']       = self::get_themes();
-			$settings['onboarding']['activeTheme']  = get_option( 'stylesheet' );
+			$settings['onboarding']['productTypes']  = self::get_allowed_product_types();
+			$settings['onboarding']['themes']        = self::get_themes();
+			$settings['onboarding']['activeTheme']   = get_option( 'stylesheet' );
+			$settings['onboarding']['activePlugins'] = self::get_active_plugins();
 		}
 
 		return $settings;
 	}
 
+	/**
+	 * Gets an array of plugins that can be installed & activated via the onboarding wizard.
+	 */
+	public static function get_allowed_plugins() {
+		return apply_filters(
+			'woocommerce_onboarding_plugins_whitelist',
+			array(
+				'jetpack'              => 'jetpack/jetpack.php',
```

Comment:
Do we have any way to catch if the folders names are mismatched?  E.g., if there's `jetpack-master/jetpack.php` I won't be able to complete the onboarding right?

Probably an edge case, but curious if we should consider a workaround in the future.

Reply:
Correct. I believe we have this same issue in the eCommerce Plan / Calypsoify when detecting installed plugins.

I could add a `@todo` since this is an edge case. We might have to get smart with some regex. I believe these directories can also sometimes end up with version numbers.

Reply:
Added a todo in https://github.com/woocommerce/woocommerce-admin/pull/2825/commits/cf79b019e0d16b50c4c31da6e702ac7caf9bf9d7.